srcpkgs/ruby-asciidoctor/template

```diff
@@ -1,13 +1,13 @@
 # Template file for 'ruby-asciidoctor'
 pkgname=ruby-asciidoctor
-version=2.0.15
-revision=5
+version=2.0.23
+revision=1
 build_style=gem
-short_desc="Ruby implementation of AsciiDoc"
-maintainer="Orphaned <orphan@voidlinux.org>"
+short_desc="Publishing toolchain of AsciiDoc, written in Ruby"
+maintainer="life4sucks <life4sucks@proton.me>"
```

Comment:
Don't adopt this package. We would like to see contributors establish a history of good package stewardship before adding their names to templates.